### PR TITLE
improve test

### PR DESCRIPTION
**Pull request by Christianlopez4**

### CODE REVIEW

File: test/google.spec.ts [hunk 1]
```diff
@@ -1,9 +1,13 @@
...
-    browser.get("http://www.google.com");
-    expect(browser.getTitle()).toEqual("Google");
+describe("Given a SDET learning protractor", () => {
+  describe("when open Google Page", () => {
+    beforeEach(() => {
+      browser.get("http://www.google.com");
+    });
+
+    it("then should have a title", () => {
+      expect(browser.getTitle()).toEqual("Google");
```
Comment (renardete) [inline test/google.spec.ts:10]: set it as a variable in case it could change in the future.

File: test/google.spec.ts [hunk 2]
```diff
@@ -1,9 +1,13 @@
...
 
-describe("This is the first example of protractor", () => {
-  it("should have a title", () => {
-    browser.ignoreSynchronization = true;
-    browser.get("http://www.google.com");
-    expect(browser.getTitle()).toEqual("Google");
+describe("Given a SDET learning protractor", () => {
+  describe("when open Google Page", () => {
+    beforeEach(() => {
+      browser.get("http://www.google.com");
```
Comment (renardete) [inline test/google.spec.ts:6]: Set it as a variable, and give it a meaningfull name

**Review by renardete — APPROVED** [review]

**Review by renardete — CHANGES_REQUESTED** [review]
update PR

File: test/google.spec.ts (hunk 2 above)
Comment (Christianlopez4) [reply test/google.spec.ts]: I already set the domain as a variable

**Review by renardete — APPROVED** [review]

**Pull request merged**